### PR TITLE
adds update task and updates documentation

### PR DESCRIPTION
**Pull request by hsingh23**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -29,10 +29,40 @@ module.exports = function(grunt) {
     },
 
     // Configuration to be run (and then tested).
-    jsdeps: {
+    update_jsdeps: {
+      // currently we can only read in json
       simple: {
+        files: {
+          expand: true,
+          // src: ["test/fixtures/update/simple/**/*.js"]
```
Comment (rickshinners) [inline Gruntfile.js]: Github code reviews are unfamiliar to me!  Did I already comment on this?

File: Gruntfile.js [hunk 2]
```diff
@@ -40,7 +70,7 @@ module.exports = function(grunt) {
       },
       relativeRoot: {
         options: {
-          root: "test/fixtures/",
+          pathPrefix: "test/fixtures/",
```
Comment (rickshinners) [inline Gruntfile.js:72]: I don't really agree with changing this.  I think we should just add documentation around what 'root' is.  pathPrefix doesn't really say what this does either.

File: tasks/jsdeps.js [hunk 3]
```diff
@@ -53,15 +53,15 @@
       }
     });
 
-    depTree[getRootRelativePath(options.root, absolutePath)] = refs
-      .map(function (x) { return getRootRelativePath(options.root, x); });
+    depTree[getPrefixRelativePath(options.pathPrefix, absolutePath)] = refs
```
Comment (rickshinners) [inline tasks/jsdeps.js:56]: Here's another spot where I think root makes more sense than prefix

File: Gruntfile.js (hunk 2 above)
Comment (hsingh23) [inline Gruntfile.js:72]: Added documentation, kept pathPrefix, because root messed me up before.

**Pull request merged**